src/meta/src/hummock/manager/mod.rs

```diff
@@ -493,3 +501,68 @@ async fn write_exclusive_cluster_id(
         }
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct TableWriteThroughputStatistic {
+    pub throughput: u64,
+    pub timestamp: i64,
```

Comment:
minor: it is better to put the unit of the timestamp in the name: `timestamp_secs`